src/rsa/public/key.rs

```diff
@@ -87,6 +98,55 @@ impl Key {
     pub fn e(&self) -> Exponent {
         self.e
     }
+
+    /// Calculates base**e (mod n), filling the first part of `out_buffer` with
+    /// the result.
+    ///
+    /// This is constant-time with respect to the value in `input` (only).
```

Comment:
Should this mention `base` instead as well?
```suggestion
    /// This is constant-time with respect to the value in `base` (only).
```